plugin/PDFContextMenuHighlight.qml

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2016 Caliste Damien.
+ * Contact: Damien Caliste <dcaliste@free.fr>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Office.PDF 1.0 as PDF
+
+ContextMenu {
+    id: contextMenuHighlight
+    property variant annotation
+
+    InfoLabel {
+        id: infoContents
+        visible: infoContents.text != ""
+        width: parent.width
+        anchors.topMargin: Theme.paddingSmall
```

Comment:
Not top anchored, so don't think this does anything. And cannot be vertically anchored anyway since ContextMenu puts this inside a Column. verticalAlignment: Text.AlignVCenter could do what's intended.